target/portfolio-1/js/createGame.js

```diff
@@ -0,0 +1,53 @@
+function addNewStage() {
+  const activeStages = document.getElementsByClassName('activeStage')[0];
+  activeStages.classList.remove('activeStage');
+  const divToHide = activeStages.classList[0];
+  document.getElementById(divToHide + 'Hints').classList.add('hidden');
+  const stagesList = document.getElementById('stages');
+  const numStages = stagesList.getElementsByTagName('input').length;
+
+  const newStage = document.createElement('input');
+  newStage.value = 'Stage ' + (numStages + 1);
+  newStage.type= 'button';
+  newStage.className = 'stage' + (numStages + 1) + ' activeStage';
```

Comment:
Use setActive() instead of this?

Reply:
This is a good idea, didn't notice I was repeating that.